cli/cmd/project/status.go

```diff
@@ -35,13 +43,18 @@ func StatusCmd(cfg *config.Config) *cobra.Command {
 
 			depl := proj.ProductionDeployment
 			if depl != nil {
+				logs, err := logsFormatter(depl.Logs)
+				if err != nil {
+					return err
+				}
```

Comment:
The logs are not guaranteed to be JSON. If the JSON parsing fails, it should just render the raw `depl.Logs` string